src/base/LMInputText/index.tsx

```diff
@@ -0,0 +1,53 @@
+import {View, Text, TextInput, StyleSheet} from 'react-native';
+import React from 'react';
+import {LMInputTextProps} from './types';
+
+const LMInputText = ({
+  inputText,
+  onType,
+  inputTextStyle,
+  placeholderText,
+  placeholderTextColor,
+  autoCapitalize,
+  keyboardType,
+  multilineField,
+  secureText,
+  disabled
+}: LMInputTextProps) => {
+  return (
+    <TextInput
+      onChangeText={val => {
```

Comment:
rename `val` to `value` or `enteredText`

Reply:
resolved